style.css

```diff
@@ -250,8 +250,8 @@ ul {
 .About {
   background-color: #fff;
   border-radius: 0 50px 0 0;
-  height: 1000px;
-  width: 375px;
+  height: 72rem;
```

Comment:
- Optional
Setting a fixed height in such a scenario is never a good idea. You can consider using `margin-bottom` on this section or `margin-top` on the Contact section to put some distance between the two.